examples/tf/dqn_cartpole.py

```diff
@@ -22,17 +22,23 @@ def run_task(*_):
         n_epoch_cycles = 10
         sampler_batch_size = 500
         num_timesteps = n_epochs * n_epoch_cycles * sampler_batch_size
+
```

Comment:
this additional spacing appears unrelated to the stated change -- please remove it